demisto_sdk/commands/create_id_set/tests/update_id_set_test.py

```diff
@@ -248,15 +248,17 @@ def test_has_duplicate():
                 'typeID': 'urlRep',
                 'fromVersion': '5.0.0',
                 'kind': 'Details',
-                'path': 'Layouts/layout-details-urlrep.json'
+                'path': 'Layouts/layout-details-urlrep.json',
+                'pack': 'urlRep1'
             }
         })
 
         id_set['Layouts'].append({
             'urlRep': {
                 'typeID': 'urlRep',
                 'kind': 'Details',
-                'path': 'Layouts/layout-details-urlrep2.json'
+                'path': 'Layouts/layout-details-urlrep2.json',
+                'pack': 'urlRep2'
             }
```

Comment:
@anara123 without the difference in the pack names, 
Should this be considered a duplicate?
Until now it was considered that way, and now it depends on the name of the pack.